ads-cert-examples/scala/README.md

```diff
@@ -0,0 +1,13 @@
+## Run
+Usage:
+```
+sbt "run [private file] [public file]"
+```
+Example:
+```
+sbt "run ../private.pem ../public.pem"
+```
+Test mismatch:
+```
+sbt "run ../private.pem ../public_bad.pem"
+```
```

Comment:
The same warning about shading the BouncyCastle JAR from Java would apply here as well.  The `sbt-assmbly` plugin is fairly popular and I can tell from unfortunate experience that extra work is needed to get it to play nice with signed JARs.